src/pages/settings/Security/MergeAccounts/MergeResultPage.tsx

```diff
@@ -228,6 +228,14 @@ function MergeResultPage() {
         };
     }, [setRootStatusBarEnabled, login, translate, userEmailOrPhone, styles]);
 
+    useEffect(() => {
```

Comment:
Can you add a comment here explaining why we need to do this (to prevent users from navigating back after the merge)?